test/Kestrel.FunctionalTests/ResponseTests.cs

```diff
@@ -2497,8 +2497,7 @@ public void ConnectionClosedWhenResponseDoesNotSatisfyMinimumDataRate()
         {
             using (StartLog(out var loggerFactory))
             {
-                var logger = loggerFactory.CreateLogger($"{typeof(ResponseTests).FullName}.{nameof(ConnectionClosedWhenResponseDoesNotSatisfyMinimumDataRate)}");
-
+                var logger = loggerFactory.CreateLogger("ConnClosedWhenRespDoesNotSatisfyMinDataRate");
```

Comment:
This is changing the logger name, not the test name used to generate the file path. You need to edit the `StartLog` call to use a shorter name rather than the `CallerMemberName` value it currently uses.